src/components/menu/menu.tsx

```diff
@@ -198,13 +206,15 @@ export class Menu {
             '--list-grid-gap',
             '--notification-badge-background-color',
             '--notification-badge-text-color',
-        ];
+        ] as const;
```

Comment:
# 😍

I didn't know about this, so had to look it up. This is awesome! I'll be using this a _ton_ from now on!

Docs for anyone else wondering: https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-4.html#const-assertions